smileidentity/web-client
# config: set up new workflows for test, deploy-preview, main

modify GitHub workflows to test and deploy-preview on PRs.
and deploy to v1 on merge to main.

to test:
1. verify that on a PR a comment with a Preview URL is shared
2. verify that a deploy to a preview deploy exists by following the URL in step 1
